src/SFA.DAS.PSRService.IntegrationTests/Web/WhenUserEntersSite.cs

```diff
@@ -26,8 +33,26 @@ public void OneTimeSetUp()
         {
             _container = new Container();
             _container.Configure(TestHelper.ConfigureIoc());
+
+            //Replace authorization service with one tailored to this test
+            //TODO: Refactor test to Geiven-Then-When style and find a better way of onfiguring authorization
+            _container.Inject<IAuthorizationService>(BuildMockAuthorizationService(new[] { PolicyNames.CanEditReport, PolicyNames.CanSubmitReport }));
         }
```

Comment:
I don't see the benefit of having an `IEnumerable<string>` argument list for this method.

It would be more intention revealing to remove the arguments and rename the method to reflect that we are mocking an `IAuthorizationSErvice` which will say any user is authorized for `CanEdit` and `CanSubmit` policies.